src/rapala.py

```diff
@@ -26,43 +44,19 @@ def __init__(
         self.article_to_start_from = article_to_start_from or 0
         self.source_to_start_from = source_to_start_from or 0
 
-        self.chrome_path = chrome_path
+        self.driver_path = driver_path 
+        self.driver_type = driver_type
 
         # disable images in browser for faster loading.
         self.prefs = prefs or {"profile.managed_default_content_settings.images": 2}
         self.driver = None
 
-        self.first_article_path = (
-            './/*[@id="content"]/div[1]/div/div/div/div[1]/div[2]/div/ul[1]/li/div/div/a/h4'
-            or "/html/body/div[2]/div/div[4]/div/div/div/div/div/div[2]/div/ul[1]/li/div/div/a/h4"
-        )
-        self.article_path = (
-            './/*[@id="ordinaryItems"]/li[{}]/div/div/a/h4'
-            or "/html/body/div[2]/div/div[4]/div/div/div/div/div/div[2]/div/ul[2]/li[{}]/div/div/a/h4"
-        )
+        self.first_article_path = CONFIG['FIRST_ARTICLE_PATH1'] or CONFIG['FIRST_ARTICLE_PATH1']
+        self.article_path = CONFIG['ARTICLE_PATH1'] or CONFIG['ARTICLE_PATH1']
```

Comment:
It's unclear why you have an `or` in both lines here since you place the same objects on both sides. You're essentially doing A or A.